pkgs/top-level/all-packages.nix

```diff
@@ -28894,5 +28894,9 @@ in
 
   lc3tools = callPackage ../development/tools/lc3tools {};
 
+  wordlists = callPackage ../tools/security/wordlists/default.nix {
+    lists = [ "dirbuster" "nmap" "rockyou" "seclists" "wfuzz" ];
+  };
```

Comment:
I think this should be a package scope, this diff shows a pretty minimal example of how to do so:
https://github.com/NixOS/nixpkgs/commit/632c4f2c9ba1f88cd5662da7bedf2ca5f0cda4a9

Reply:
and then make it so that wordlists exposes a `withLists` helper similar to python's `withPackages` function:
```
  wordlists.withLists(l: with l; [ dirbuster nmap rockyou seclists wfuzz ])
```

also avoids things being "stringly" typed

Reply:
the `withLists` can be implemented in a later PR, just demonstrating why a package scope would be beneficial

Reply:
Hmmm, I am not too sure how that would work...

Especially with the script that is currently created (to create the `wordlists` and `wordlists_path` commands).

From the example you shared, and reading more in the current `nixpkgs`, I have the impression that `cinnamon` is "just" a bundle of packages.
Right now, I was seeing `wordlists` as a 1) Bundle of wordlists, 2) Handy commands to access them.

Reply:
@jonringer, I am really not familiar with this pattern...

I kinda refactored the original code (first commit); And tried to follow your hint, although I kept it separate (third commit).
Is that the kind of things you were hoping to see?

Now I am having a look on how to implement some `withLists`...
And probably have the `extraLists` feature to follow a similar idea.